src/schema/v2/me/myCollectionUpdateArtworkMutation.ts

```diff
@@ -219,13 +219,15 @@ export const myCollectionUpdateArtworkMutation = mutationWithClientMutationId<
 
       const imageSources = computeImageSources(externalImageUrls)
 
+      const newImages: any[] = []
       for (const imageSource of imageSources) {
-        await createArtworkImageLoader(artworkId, imageSource)
+        newImages.push(await createArtworkImageLoader(artworkId, imageSource))
       }
 
       return {
         ...response,
         id: artworkId,
+        images: [...(response?.images || []), ...newImages],
```

Comment:
An alternative might be, after all of the updates and _before_ this `return`, to do an `artworkLoader(artworkId)` call (basically fetch the newly created artwork). This might be more accurate, as it should (hopefully!) include all images and whatever other things have been created/set.

Reply:
Very good idea 👏 I've updated the resolvers for both My Collection artwork mutations (create & update).

Reply:
nice idea